packages/next/package.json

```diff
@@ -78,13 +81,17 @@
     "crypto-browserify": "3.12.0",
     "css-loader": "4.3.0",
     "cssnano-simple": "1.2.1",
+    "eslint": "7.9.0",
+    "eslint-plugin-import": "2.22.1",
+    "eslint-plugin-react-hooks": "4.1.2",
```

Comment:
Can we handle these dependencies the same way we did for `critters` [here](https://github.com/vercel/next.js/pull/16539/files) making them installed on the mono-repo and not on the `next package itself? It seems these increase install size by 20mb which we'll want to avoid while this is experimental

Reply:
totally, just discussed with @janicklas-ralph .